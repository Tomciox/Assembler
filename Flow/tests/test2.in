2 3 0.1
1 2
1 2
1 2
1 
0.5 0.5 0.5