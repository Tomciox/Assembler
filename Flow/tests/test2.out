0.900000 1.800000 
0.850000 1.700000 
0.900000 1.800000 

